source/Patches/Roles/Role.cs

```diff
@@ -512,6 +512,45 @@ public static class HudManager_Update
             private static Vector3 oldScale = Vector3.zero;
             private static Vector3 oldPosition = Vector3.zero;
 
+            //add the colorType List from DeadBody.cs of MedicMod folder to use here
+            public static String GetColorType(PlayerControl player)
+            {
+                var colors = new Dictionary<int, string>
+                {
+                    {0, "darker"},// red
```

Comment:
Let's replace "darker" and "lighter" with an enum (or possibly a boolean, but enum feels cleaner).

Reply:
i'm ok, i've just take the original code

Reply:
Yeah, but the original code is bad. Let's make it better.

Reply:
Ok, Can you explain me quickly what you wanna for that ?

Reply:
Replace the strings "darker" and "lighter" with an enum, and return that enum instead of returning a string. Then instead of comparing against the words "lighter" and "darker", compare against the enum.

Using strings, especially strings that aren't in a constant, are really brittle. It's easy to make a typo or something and get it wrong. Using an enum is cleaner and less error-prone.

Reply:
ok so I keep the dictionary, but I replace the strings with an enum like ColorType.Darker or ColorType.Lighter?

Reply:
Yes